src/db/models/enrollment.rs

```diff
@@ -376,13 +392,20 @@ impl User {
         mail_tx: UnboundedSender<Mail>,
     ) -> Result<String, TokenError> {
         info!(
-            "User {} starting enrollment for user {}, notification enabled: {send_user_notification}",
-            admin.username, self.username
+            "Start generating a new enrollment process for user {}.",
+            self.username
+        );
```

Comment:
Pisanie kto zaczął enrollment tutaj jest okej

```suggestion
            "User {} started a new enrollment process for user {}.",
            admin.username, self.username
        );
```